pynfe/processamento/serializacao.py

```diff
@@ -190,7 +192,7 @@ def _serializar_cliente(
             etree.SubElement(raiz, "indIEDest").text = "9"
         elif (
             cliente.indicador_ie == 2 or cliente.isento_icms
-        ) or cliente.inscricao_estadual.upper() == "ISENTO":
+        ) or (cliente.inscricao_estadual or '').upper() == "ISENTO":
```

Comment:
Acho que o certo é colocar um Throw se tentar colocar o inscricao_estadual como None OU no __init__ da classe preencher ele como "" caso for None.